cli/run_util_test.go

```diff
@@ -607,3 +607,108 @@ func TestAddTestMetadata(t *testing.T) {
 		}
 	}
 }
+
+func Test_activateStepLibStep(t *testing.T) {
+	tests := []struct {
+		name        string
+		stepIDData  models.StepIDData
+		wantVersion string
+		wantErr     bool
+	}{
```

Comment:
Can you add a test case for "" and x.x.x (even if not supported, just as a reference)